.github/workflows/release.yml

```diff
@@ -98,10 +98,10 @@ jobs:
         uses: actions/checkout@v4
         with:
           ref: "${{ needs.create-tag.outputs.release-tag }}"
-      - name: Set up Python ${{ env.default-python }}
+      - name: Set up Python ${{ env.minimum-supported-python }}
```

Comment:
This env variable is not yet defined in this workflow.

Reply:
Good catch! Fixed.